src/swap_config.jl

```diff
@@ -0,0 +1,91 @@
+# Not a module, just supplementary code for the swap_config function, this is essential given the inclusion of the new energy types.
+
+"""
+    swap_config!(mc_state,movetype::atommove)
+    swap_config!(mc_state,movetype::volumemove)
+
```

Comment:
at present there is only one method implemented; but it looks like that the swap_config! is not needed and one could have different methods depending on move type of swap_config that are the swap_atom_config and swp_config_v